docs/api/RESTAPI.md

```diff
@@ -283,7 +283,7 @@ Query for workflows which match various criteria. When a combination of criteria
 |Type|Name|Description|Schema|Default|
 |---|---|---|---|---|
 |**Path**|**version**  <br>*required*|Cromwell API Version|string|`"v1"`|
-|**Query**|**additionalQueryResultFields**  <br>*optional*|Currently only 'labels' is a valid value here. Use it to include a list of labels with each result.|< string > array(multi)||
+|**Query**|**additionalQueryResultFields**  <br>*optional*|Currently only 'labels' and `parentworkflowId` are valid values here. Use it to include a list of labels and/or parent workflow IDs with each result.|< string > array(multi)||
```

Comment:
Is that capitalization really correct for `parentworkflowId `...? 😨

Reply:
In the other file this is written as `parentWorkflowId `

Reply:
@cjllanwarne Yeah duh. I just did a global search for text files which had this, forgot about the autogeneration